OpenRA.Mods.Common/Activities/Move/Drag.cs

```diff
@@ -19,7 +19,7 @@ public class Drag : Activity
 	{
 		readonly IPositionable positionable;
 		readonly IMove movement;
-		readonly IDisableMove[] moveDisablers;
+		readonly IDisabledTrait disableable;
```

Comment:
When I add locomotor logic, there will be a trait (`Positionable/Mobile`) which will manage the locomotors including route-finding over different terrain using one or more locomotors.